src/ray/gcs/redis_context.cc

```diff
@@ -135,7 +135,30 @@ RedisContext::~RedisContext() {
   }
 }
 
-Status RedisContext::Connect(const std::string &address, int port, bool sharding) {
+static std::string default_password = "";
```

Comment:
This appears in multiple places. Consider moving it to https://github.com/ray-project/ray/blob/master/src/ray/constants.h.